common.blocks/popup/popup.tests/gemini.blocks/test-nested/test.styl

```diff
@@ -1 +0,0 @@
-../../simple.blocks/test/test.styl
```

Comment:
This "feature" of the `enb-bem-examples`: create symlinks, but does not change contents of files.